app/src/main/java/com/github/geohunt/app/ui/components/statistics/DateGraph.kt

```diff
@@ -0,0 +1,65 @@
+package com.github.geohunt.app.ui.components.statistics
+
+import androidx.compose.runtime.Composable
+import com.github.geohunt.app.i18n.DateFormatUtils
+import com.github.geohunt.app.ui.components.utils.Graph
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+/**
+ * A graph that draws x/y points, where x is a date and y is a value
+ * @param xDateValues The x values of our points represented by dates
+ *  have to be in the range dateGranularity.dateRange(LocalDate.now())
+ * @param dateGranularity the DateGranularity of the graph, gives the range of dates that will
+ *  be shown on the graph (last week, month, year)
+ * @param yValues the y values of our points represented by Long values
+ */
+@Composable
+fun DateGraph(
```

Comment:
I think you should allow the caller of the function to determine the date for which the DateGraph is going to be displayed. The code of the function should like that in my opinion:

`fun DateGraph(
        xDateValues: List<LocalDate>,
        dateGranularity: DateGranularity,
        yValues: List<Long>,
        currentDate: LocalDate = LocalDate.now(),
) {
    ...
}`